### PR TITLE
Switch default branch to `staging` for template tests

### PR DESCRIPTION
**Pull request by weikanglim**
We want our scheduled test runs to run against template repository `staging` by default using azd `daily`. This was changed inadvertently when refactoring the template tests.

### CODE REVIEW

File: eng/pipelines/template-tests.yml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+pr:
+  paths:
+    include: 
+      - eng/templates/**/*template-test*.yml
+      - eng/pipelines/template-tests.yml
```
Comment (wbreza) [inline eng/pipelines/template-tests.yml:5]: Will this cause the pipeline to only change on PR's that include this change or will the scheduled run also continue to run as planned?

**Review by wbreza — APPROVED** [review]
One question, otherwise looks good.

File: eng/pipelines/template-tests.yml (hunk 1 above)
Comment (danieljurek) [reply eng/pipelines/template-tests.yml:5]: This will not affect scheduled runs. Scheduled runs are configured manually in the Azure DevOps UI.

**Review by danieljurek — APPROVED** [review]

**Pull request merged**